examples/flask-hello-world-pyproject/.gitignore

```diff
@@ -0,0 +1 @@
+*.egg-info
```

Comment:
Do we want to add this to the generated .gitignore? or is there a way to put this artefact somewhere in .pixi/ ?

Reply:
@tdejager any thought on this?

Reply:
I dont think there is a way its a `setuptools` thing, so lets just add it to the .gitignore.

Reply:
Done. I have added it to the default gitignore template rather than adding conditional around 1/the type of manifest and 2/ testing for the presence of `*.egg-info` in the gitignore file.

Let me know if you feel that added logic is worth it.